### PR TITLE
Update crystal shards location

### PR DESCRIPTION
**Pull request by jwaldrip**
CrystalShards.xyz is down once again. CrystalShards.org is more stable and has not experienced downtime in the last few years.

### CODE REVIEW

**Comment by straight-shoota** [comment]
I think we should probably not just mention a single website there. Not sure who even declared that to be "the go-to place" 🤷 
Maybe a link to https://crystal-lang.org/community/#shards would be the best idea. There we already link to several options and there's no point in maintaining two lists.

**Comment by jwaldrip** [comment]
@straight-shoota perfect!

**Comment by jwaldrip** [comment]
@straight-shoota you may want to drop xyz off the community site as well. And would love if you could add org to it!

File: docs/guides/writing_shards.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ A description should be:
 #### Optimizing
 
 It's hard for anyone to use your project if they can't find it.
-[crystalshards.xyz](https://crystalshards.xyz/) is currently the go-to place for Crystal libraries, so that's what we'll optimize for.
+Take a look [here](https://crystal-lang.org/community/#shards) for more information on where to find shards
```
Comment (straight-shoota) [inline docs/guides/writing_shards.md:163]: I despise "here" links.
```suggestion
There are several services for discovering shards, a list is available on the [Crystal Community page](https://crystal-lang.org/community/#shards).
```

**Comment by jwaldrip** [comment]
Thanks @straight-shoota I opened a PR to the community page as well on the crystal-website repo.

**Review by straight-shoota — APPROVED** [review]

**Review by oprypin — APPROVED** [review]

**Pull request merged**